portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,12 +38,14 @@
 public class DataServlet extends HttpServlet {
   
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private Query query;
+  private PreparedQuery results;
```

Comment:
What's the reasonings/tradeoffs for keeping these as class vars as opposed to initializing the vars in the methods themselves?

Reply:
I thought that it was being referenced by different methods so having a class reference would be better than to have it in the method.

Reply:
I thought about it and i think making those variables local makes more sense.